src/App.tsx

```diff
@@ -1,18 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [page, onPageChange] = useState(1);
+  const [perPage, onPerPageChange] = useState(5);
```

Comment:
```suggestion
  const [page, setPage] = useState(1);
  const [perPage, setPerPage] = useState(5);
```
onSmthChange то зазвичай якісь слухачі подій, а тут просто сетиться значення